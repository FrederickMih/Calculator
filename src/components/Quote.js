import React from 'react';
import '../styles/Quotes.css';

const Quote = () => (
  <div className="some-quotes">
    <h3 className="quotes-h3">Quotes</h3>
    <p>
      <i>
        {' '}
        Mathematics is not about numbers, equations, computations, or
        algorithms: it is about understanding. –William Paul Thurston
      </i>
    </p>
    <br />
    <p>
      <i>
        {' '}
        Pure mathematics is, in its way, the poetry of logical ideas. — Albert
        Einstein, German theoretical physicist
      </i>
    </p>
    <br />
    <p>
      <i>
        {' '}
        Mathematics allows for no hypocrisy and no vagueness. — Stendhal (pen
        name of Marie-Henri Beyle), French writer
      </i>
    </p>
  </div>
);

export default Quote;
