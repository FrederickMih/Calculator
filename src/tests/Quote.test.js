import React from 'react';
import renderer from 'react-test-renderer';
import { render, screen } from '@testing-library/react';
import Quote from '../components/Quote';
import '@testing-library/jest-dom';

it('should renders correctly to Quote component', () => {
  const tree = renderer.create(<Quote />).toJSON();
  expect(tree).toMatchSnapshot();
});

it('doest not contain Welcome!', () => {
  render(<Quote />);
  expect(screen.queryByText('Welcome!')).not.toBeInTheDocument();
});
