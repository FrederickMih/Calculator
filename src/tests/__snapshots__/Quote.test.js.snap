// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should renders correctly to Q 1`] = `
<div
  className="some-quotes"
>
  <h3
    className="quotes-h3"
  >
    Quotes
  </h3>
  <p>
    <i>
       
      It’s fine to work on any problem, so long as it generates interesting mathematics along the way — even if you don’t solve it at the end of the day. — Andrew Wiles
    </i>
  </p>
  <br />
  <p>
    <i>
       
      Pure mathematics is, in its way, the poetry of logical ideas. — Albert Einstein, German theoretical physicist
    </i>
  </p>
  <br />
  <p>
    <i>
       
      Mathematics allows for no hypocrisy and no vagueness. — Stendhal (pen name of Marie-Henri Beyle), French writer
    </i>
  </p>
  <br />
  <p>
    <i>
       
      Mathematics may not teach us to add love or subtract hate, but it gives us hope that every problem has a solution. — Anonymous
    </i>
  </p>
</div>
`;

exports[`should renders correctly to Quote component 1`] = `
<div
  className="some-quotes"
>
  <h3
    className="quotes-h3"
  >
    Quotes
  </h3>
  <p>
    <i>
       
      It’s fine to work on any problem, so long as it generates interesting mathematics along the way — even if you don’t solve it at the end of the day. — Andrew Wiles
    </i>
  </p>
  <br />
  <p>
    <i>
       
      Pure mathematics is, in its way, the poetry of logical ideas. — Albert Einstein, German theoretical physicist
    </i>
  </p>
  <br />
  <p>
    <i>
       
      Mathematics allows for no hypocrisy and no vagueness. — Stendhal (pen name of Marie-Henri Beyle), French writer
    </i>
  </p>
  <br />
  <p>
    <i>
       
      Mathematics may not teach us to add love or subtract hate, but it gives us hope that every problem has a solution. — Anonymous
    </i>
  </p>
</div>
`;

exports[`should renders correctly to Quote page 1`] = `
<div
  className="some-quotes"
>
  <h3
    className="quotes-h3"
  >
    Quotes
  </h3>
  <p>
    <i>
       
      It’s fine to work on any problem, so long as it generates interesting mathematics along the way — even if you don’t solve it at the end of the day. — Andrew Wiles
    </i>
  </p>
  <br />
  <p>
    <i>
       
      Pure mathematics is, in its way, the poetry of logical ideas. — Albert Einstein, German theoretical physicist
    </i>
  </p>
  <br />
  <p>
    <i>
       
      Mathematics allows for no hypocrisy and no vagueness. — Stendhal (pen name of Marie-Henri Beyle), French writer
    </i>
  </p>
  <br />
  <p>
    <i>
       
      Mathematics may not teach us to add love or subtract hate, but it gives us hope that every problem has a solution. — Anonymous
    </i>
  </p>
</div>
`;

exports[`should renders correctly to compone 1`] = `
<div
  className="some-quotes"
>
  <h3
    className="quotes-h3"
  >
    Quotes
  </h3>
  <p>
    <i>
       
      It’s fine to work on any problem, so long as it generates interesting mathematics along the way — even if you don’t solve it at the end of the day. — Andrew Wiles
    </i>
  </p>
  <br />
  <p>
    <i>
       
      Pure mathematics is, in its way, the poetry of logical ideas. — Albert Einstein, German theoretical physicist
    </i>
  </p>
  <br />
  <p>
    <i>
       
      Mathematics allows for no hypocrisy and no vagueness. — Stendhal (pen name of Marie-Henri Beyle), French writer
    </i>
  </p>
  <br />
  <p>
    <i>
       
      Mathematics may not teach us to add love or subtract hate, but it gives us hope that every problem has a solution. — Anonymous
    </i>
  </p>
</div>
`;

exports[`sould renders correctly to Quote page 1`] = `
<div
  className="some-quotes"
>
  <h3
    className="quotes-h3"
  >
    Quotes
  </h3>
  <p>
    <i>
       
      It’s fine to work on any problem, so long as it generates interesting mathematics along the way — even if you don’t solve it at the end of the day. — Andrew Wiles
    </i>
  </p>
  <br />
  <p>
    <i>
       
      Pure mathematics is, in its way, the poetry of logical ideas. — Albert Einstein, German theoretical physicist
    </i>
  </p>
  <br />
  <p>
    <i>
       
      Mathematics allows for no hypocrisy and no vagueness. — Stendhal (pen name of Marie-Henri Beyle), French writer
    </i>
  </p>
  <br />
  <p>
    <i>
       
      Mathematics may not teach us to add love or subtract hate, but it gives us hope that every problem has a solution. — Anonymous
    </i>
  </p>
</div>
`;
